ocrd_lib.sh

```diff
@@ -29,13 +36,25 @@ init() {
     exit 2
   fi
 
+  WORKDIR=ocr-d/"$PROCESS_DIR" # use subdirectory of same volume so --reflink CoW still possible
+  if ! mkdir -p "$WORKDIR"; then
+    logger -p user.error -t $TASK "insufficient permissions on /data volume"
+    exit 5
+  fi
+  # try to be unique here (to avoid clashes)
+  REMOTEDIR="KitodoJob_${PID}_$(basename $PROCESS_DIR)"
+
   WORKFLOW=$(command -v "$WORKFLOW" || realpath "$WORKFLOW")
   if ! test -f "$WORKFLOW"; then
     logger -p user.error -t $TASK "invalid workflow '$WORKFLOW'"
     exit 3
   fi
   logger -p user.notice -t $TASK "using workflow '$WORKFLOW':"
   ocrd_format_workflow | logger -p user.notice -t $TASK
+  if test "${WORKFLOW#/workflows/}" = "$WORKFLOW"; then
```

Comment:
So i think we should add this as hint for future development

```suggestion
  # The check trivially looks whether the realpath of the workflow starts with /workflows and thus, needs no further action. If it does not, then it gets copied into the workdir under a default name, to ensure it will be accessible.
  if test "${WORKFLOW#/workflows/}" = "$WORKFLOW"; then
```

Reply:
No. I just paraphrased what the code does. Copying my description verbatim makes no sense. I don't think the line needs a comment at all. If it does, it should be an actual comment.